### PR TITLE
Add bootstrap spinner

### PR DESCRIPTION
**Pull request by itzmeowww**

### CODE REVIEW

**Comment by Omkaragrawal** [comment]
Hey, I am glad you took up the task to add a bootstrap spinner. But unfortunately, you have changed the formatting a lot. I would recommend reformating the code in the format as before, where the HTML tag's properties are not on separate lines and the extra few lines are added in between the HTML tag.

@itzmeowww

**Comment by itzmeowww** [comment]
@Omkaragrawal I have made some changes, please check it

File: index.html [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; img-src 'self' data:; connect-src 'self' https://hacker-news.firebaseio.com https://corsenabled.herokuapp.com  https://*.hackerearth.com;script-src 'self' https://cdnjs.cloudflare.com; style-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com;">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self'; img-src 'self' data:; connect-src 'self' https://hacker-news.firebaseio.com https://corsenabled.herokuapp.com  https://*.hackerearth.com;script-src 'self' https://cdnjs.cloudflare.com; style-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com;"
+    />
```
Comment (Omkaragrawal) [inline index.html]:
```suggestion
    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; img-src 'self' data:; connect-src 'self' https://hacker-news.firebaseio.com https://corsenabled.herokuapp.com  https://*.hackerearth.com;script-src 'self' https://cdnjs.cloudflare.com; style-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com;" />
```

**Review by Omkaragrawal — CHANGES_REQUESTED** [review]
I marked on last change that I want from you and then you are good to go.

File: index.html (hunk 1 above)
Comment (Omkaragrawal) [reply index.html]: @itzmeowww

**Pull request merged**